docs/jwst/badpix_selfcal/arguments.rst

```diff
@@ -0,0 +1,14 @@
+Step Arguments
+==============
+The ``badpix_selfcal`` step has the following optional arguments.
+
+``--flagfrac`` (float, default=0.001)
+  The fraction of pixels to flag as outliers on each of the low-flux and high-flux
+  sides of the smoothed-subtracted image.
```

Comment:
Docs need updating for the new flagfrac_lower and flagfrac_upper parameters.  (Thanks for those!)